ipatests/test_integration/test_sso.py

```diff
@@ -0,0 +1,100 @@
+from __future__ import absolute_import
+
+import textwrap
+from ipatests.test_integration.base import IntegrationTest
+from ipatests.pytest_ipa.integration import tasks, create_keycloak
+from ipatests.pytest_ipa.integration import create_bridge
+
+user_code_script = textwrap.dedent("""
+from selenium import webdriver
+from datetime import datetime
+from selenium.webdriver.firefox.options import Options
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+options = Options()
+options.headless = True
+driver = webdriver.Firefox(executable_path="/opt/geckodriver", options=options)
+verification_uri = "https://{hostname}:8443/auth/realms/master/account/#/"
+driver.get(verification_uri)
+
+try:
+    element = WebDriverWait(driver, 90).until(
+        EC.element_to_be_clickable((By.ID, "landingSignInButton")))
+    driver.find_element(By.ID, "landingSignInButton").click()
+    element = WebDriverWait(driver, 90).until(
+        EC.presence_of_element_located((By.ID, "kc-login")))
+    driver.find_element(By.ID, "username").send_keys("{username}")
+    driver.find_element(By.ID, "password").send_keys("{password}")
+    driver.find_element(By.ID, "kc-login").click()
+    element = WebDriverWait(driver, 900).until(
+        EC.text_to_be_present_in_element((By.ID, "landingLoggedInUser"),
+                                         "{username_fl}"))
+    assert driver.find_element(By.ID, "landingLoggedInUser").text \
+        == "{username_fl}"
+finally:
+    now = datetime.now().strftime("%M-%S")
+    driver.get_screenshot_as_file("/tmp/selenium-screenshot-%s.png" % now)
```

Comment:
Could you please update it to `/var/log/httpd/`
In artifacts it will be listed then for debugging purpose.

Reply:
changed from tmp to /var/log/httpd.